cosmos/providers/dbt/dag.py

```diff
@@ -1,13 +1,16 @@
 """
 This module contains a function to render a dbt project as an Airflow DAG.
 """
-from typing import Any, Literal
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
```

Comment:
do we need to add `typing_extensions` to the requirements?

Reply:
`typing_extensions` is [installed](https://github.com/apache/airflow/blob/main/setup.cfg#L137) with `apache-airflow` which is already a dependency in the pyproject.toml. I'd be happy to add it here too if you think that's a good idea.

Reply:
Hmm, let's add it just to be explicit & safe if you don't mind